src/Render/TagHooks.php

```diff
@@ -23,6 +24,12 @@
  * This class is responsible for rendering tags.
  */
 class TagHooks {
+
+	/**
+	 * @var bool|null
+	 */
+	private $officialForm;
```

Comment:
Why would you use a class member variable here? It is only used in one function and should therefore be a local variable.